openadapt/app/util.py

```diff
@@ -6,7 +21,12 @@
 from openadapt.scripts.reset_db import reset_db
 
 
-def clear_db(log=None):
+def clear_db(log: Any = None) -> None:
```

Comment:
`log: openadapt.app.objects.console.Console`

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/ea5e2a6ac19b1a3ea2ce9b5c89fb410dfa0bb171